src/foam/java/refinements.js

```diff
@@ -334,7 +334,8 @@ foam.CLASS({
       };
 
       // set value
-      setter += `boolean oldIsSet = ${this.name}IsSet_;\n`;
+      // Remove the useless assignment to local variable
```

Comment:
remove the commented lines please.  The comment is in your commit - presuming.

Reply:
Yes, just remove.